imap_processing/lo/l0/compression_tables.py

```diff
@@ -0,0 +1,853 @@
+"""Compression tables for decompompressing Lo L0 data."""
```

Comment:
I feel like this could be a class that you import to use/call.

Reply:
You mean each table could be a class instead?